README.md

```diff
@@ -21,6 +21,7 @@ Build Truffle artifacts:
 
 ```sh
 yarn build
+yarn copy-artifacts
```

Comment:
Also, while I think it currently shouldn't matter because the artifacts from `dex-contracts` should have the various network addresses injected already(?), wouldn't it be better to copy the artifacts before `networks-inject` (which is part of `build`)?

Reply:
I thought about making it part of the build process. But there is an issue; imagine you have already deployed it against ganache and then you change the FleetDeployment contract and hence want to rebuild it. This would erase your current migration with the addresses, as this copy command would overwrite your current artifact.

But you are probably saying that this is an edge case that is very unlikely to happen as a contract change will most likely always require a complete immigration of the contracts. Hm.

Reply:
The reason I suggest to merge it with `yarn build` is that it's a useless step for most of our users, since they would possibly don't even know what Ganache is, while the very few people who would want to use Ganache probably know how to read a `package.json`.
In case you want to keep the two separate anyway, the tutorial needs to be updated!